File: ProyectoTIC_WS/controller/ingresarEvento.php
<?php
    require_once "../bd/Data.php";

    $id_ciudad=$_GET["id_ciudad"];
    $id_usuario=$_GET["id_usuario"];
    $inicio_evento=$_GET["inicio"];
    $termino_evento=$_GET["termino"];
    $direccion=$_GET["direccion"];
    $cantidad_personal=$_GET["cantidad_personal"];

    $d= new Data();
    $d ->IngresarEvento($id_ciudad, $id_usuario, $inicio_evento, $termino_evento, $direccion, $cantidad_personal);
?>

File: ProyectoTic/src/services/IngresarEvento.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package services;

/**
 *
 * @author Emilio
 */
public class IngresarEvento {
    private String url;
    private String controller = "ingresarEvento.php";
    
    public IngresarEvento(
            String server, 
            int id_ciudad, 
            int id_usuario, 
            String inicio, 
            String termino, 
            String direccion, 
            int cantidad_personal
    )
    {
        this.url = server+"/"+controller+"?"
                + "id_ciudad="+id_ciudad+"&"
                + "id_usuario="+id_usuario+"&"
                + "inicio="+inicio+"&"
                + "termino="+termino+"&"
                + "cantidad_personal="+cantidad_personal+"&"             
                + "direccion="+direccion;
    }
    
    public String getUrl() {
        return url;
    }       
}
